src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -24,10 +24,10 @@ interface ContainerInfo {
 interface WorkloadPodLogsState {
   containerInfo?: ContainerInfo;
   duration: DurationInSeconds;
-  loadingPodLogs: boolean;
-  loadingContainerLogs: boolean;
-  loadingPodLogsError?: string;
-  loadingContainerLogsError?: string;
+  loadingAppLogs: boolean;
+  loadingProxyLogs: boolean;
+  loadingAppLogsError?: string;
+  loadingProxyLogsError?: string;
   podValue?: number;
   podLogs?: PodLogs;
   containerLogs?: PodLogs;
```

Comment:
similar name changes would be good for consistency.

Reply:
ok